logging/log_collection_forwarding/log-forwarding.adoc

```diff
@@ -7,6 +7,71 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
+The Cluster Logging Operator deploys a collector based on the `ClusterLogForwarder` resource specification. There are two collector options supported by this Operator: the legacy Fluentd collector, and the Vector collector.
+
+include::snippets/logging-fluentd-dep-snip.adoc[]
+
+[id="log-forwarding-collection"]
+== Log collection
+
+link:https://vector.dev/docs/about/what-is-vector/[Vector] is a log collector offered as an alternative to Fluentd for the {logging}.
+
+You can configure which logging collector type your cluster uses by modifying the `ClusterLogging` custom resource (CR) `collection` spec.
+
+[IMPORTANT]
+====
+Vector does not support FIPS Enabled Clusters.
+====
+
+.Example ClusterLogging CR that configures Vector as the collector
+[source,yaml]
+----
+  apiVersion: "logging.openshift.io/v1"
+  kind: ClusterLogging
+  metadata:
+    name: instance
+    namespace: openshift-logging
+    annotations:
+      logging.openshift.io/preview-vector-collector: enabled
+  spec:
+    collection:
+    logs:
+      type: vector
+      vector: {}
+----
+
+include::modules/logging-vector-fluentd-feature-comparison.adoc[leveloffset=+2]
+
+include::cluster-logging-about-collector.adoc[leveloffset=+2]
+
+[id="log-forwarding-collector-outputs"]
+=== Collector outputs
+
+The following outputs are supported:
+
+* `elasticsearch`. An external Elasticsearch instance. The `elasticsearch` output can use a TLS connection.
+
+* `kafka`. A Kafka broker. The `kafka` output can use an unsecured or TLS connection.
+
+* `loki`. Loki, a horizontally scalable, highly available, multitenant log aggregation system.
+
+.Outputs
+[options="header"]
+|==========================================================
+| Feature              | Fluentd  | Vector
+| Elasticsearch v5-v7  | &#10003; | &#10003;
```

Comment:
This should be Elasticsearch v6-v8

Reply:
Google Cloud Logging is supported from 5.5.1 https://issues.redhat.com/browse/LOG-2574